dist/drives.js

```diff
@@ -13,6 +13,9 @@ export function getRoutesSegments(dongleId, start, end, limit) {
     limit
   });
 }
+export function getRouteSegmentsById(dongleId, routeId) {
+  return StagingRequest.get(`v1/devices/${dongleId}/routes_segments?route_str=${`${dongleId}|${routeId}`.replace(/%7C/g, '|')}`);
```

Comment:
```suggestion
  return request.get(`v1/devices/${dongleId}/routes_segments?route_str=${`${dongleId}|${routeId}`.replace(/%7C/g, '|')}`);
```

can we just pass in routeName like below?